### PR TITLE
Feature/stage3

### PR DESCRIPTION
**Pull request by annammarri**
Please check this stage3

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
+//Framework extensions
 import { Component } from '@angular/core';
 import { Platform } from 'ionic-angular';
 import { StatusBar, Splashscreen } from 'ionic-native';
-
+//Pages
```
Comment (elizondo1288) [inline src/app/app.component.ts:5]: Good to see the change here

File: src/app/app.module.ts [hunk 2]
```diff
@@ -20,9 +28,11 @@ import { Storage } from '@ionic/storage';
...
     MyApp,
     HomePage,
-    AddRestaurantPagePage
+    AddRestaurantPagePage,
+    DetailsPage
   ],
   providers: [{provide: ErrorHandler, useClass: IonicErrorHandler},
-              Storage]
+              Storage,
+              RestaurantService]
```
Comment (elizondo1288) [inline src/app/app.module.ts:36]: ok now it's a singleton, good

File: src/pages/details/details.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  </ion-navbar>
+</ion-header>
+
+<ion-content padding>
+  <div class="img-position">
+    <img src="../assets/img/{{type}}.jpg">
+    <div class="restaurant-title">{{name}}</div>
+    <ion-list>
+      <ion-item>
+        <ion-label fixed>Type: </ion-label>
```
Comment (elizondo1288) [inline src/pages/details/details.html:13]: why fixed? what happens if you remove the fix label?

Comment (annammarri) [reply src/pages/details/details.html:13]: that's to align the label next to the input box

File: src/app/app.component.ts (hunk 1 above)
Comment (annammarri) [reply src/app/app.component.ts:5]: thanks

File: src/app/app.module.ts (hunk 2 above)
Comment (annammarri) [reply src/app/app.module.ts:36]: thanks

**Review by elizondo1288 — APPROVED** [review]
Looks good

**Pull request merged**